test/integration/util/util.go

```diff
@@ -41,6 +42,10 @@ type OcRunner struct {
 	CommandPath string
 }
 
+const (
+	OpenShiftVersion = "v1.4.1" // Should match OPENSHIFT_VERSION in Makefile
```

Comment:
-1, this creates dependencies which will be hard to track and lead to failing tests as we use different versions. It also prevents us from running the test against other versions of OpenShift.